core/trino-main/src/main/java/io/trino/SystemSessionProperties.java

```diff
@@ -298,6 +299,11 @@ public SystemSessionProperties(
                         "Scale out writers based on throughput (use minimum necessary)",
                         featuresConfig.isScaleWriters(),
                         false),
+                integerProperty(
+                        MAX_WRITER_TASKS_COUNT,
+                        "Set upper limit on number of tasks that will take part in writing data",
```

Comment:
Make it similar to config description: `Maximum number of tasks that will take part in writing data`